epersons.md

```diff
@@ -2,10 +2,54 @@
 [Back to the list of all defined endpoints](endpoints.md)
 
 ## Main Endpoint
-**/api/eperson/epersons**  
+**GET /api/eperson/epersons**
 
 ## Single EPerson
-**/api/eperson/epersons/<:uuid>**
+**GET /api/eperson/epersons/<:uuid>**
+
+```json
+{
+  "id": "028dcbb8-0da2-4122-a0ea-254be49ca107",
+  "uuid": "028dcbb8-0da2-4122-a0ea-254be49ca107",
+  "name": "user@institution.edu",
+  "handle": null,
+  "metadata": [
+    {
+      "key": "eperson.firstname",
+      "value": "John",
+      "language": null
+    },
+    {
+      "key": "eperson.lastname",
+      "value": "Doe",
+      "language": null
+    },
+    {
+      "key": "eperson.phone",
+      "value": "",
+      "language": null
+    },
+    {
+      "key": "eperson.language",
+      "value": "en",
+      "language": null
+    }
+  ],
+  "netid": null,
+  "lastActive": null,
+  "canLogIn": true,
+  "email": "user@institution.edu",
+  "requireCertificate": false,
+  "selfRegistered": true,
+  "groups": null,
+  "type": "eperson",
+  "_links": {
+    "self": {
+      "href": "https://dspace7-internal.atmire.com/rest/api/eperson/epersons/028dcbb8-0da2-4122-a0ea-254be49ca107"
```

Comment:
can you supply an example from the official REST demo? what about https://dspace7.4science.it/dspace-spring-rest/#https://dspace7.4science.it/dspace-spring-rest/api/eperson/epersons/15508d07-d9ee-4c42-acbd-72fe9a013504

Reply:
This was solved